### ISSUE TITLE
[API v3] Current week

### ISSUE THREAD

**Issue by techguydave** [the-blue-alliance/the-blue-alliance#1423]
Expose the current week in the status endpoint of the API. Possibly even include week numbers in the individual event endpoint responses.

Thinking about it even more, maybe we could also add an `/events/<year>/<week>` or similar extension to the events listing to only grab the current week's events.

**Comment by phil-lopreiato** [comment]
Take a look at #1107, it does most of what you're looking for, I think. We just haven't decided to ship it yet

**Comment by synth3tk** [comment]
I didn't even think to see if that PR was relevant. Thanks.

**Comment by synth3tk** [comment]
Re-opening, as I realized that PR only covers the second part. It'd still be good to have the week number in the status.

**Comment by fangeugene** [comment]
Done in APIv3

**Comment by bdaroz** [comment]
Need to reopen this one... It's in spec, but not in the endpoint, so unfortunately, not done.

**Comment by synth3tk** [comment]
Yeah, not quite sure why this was closed. I'm not seeing where it was implemented anywhere.

**Comment by bdaroz** [comment]
Removed stale tag. Probably useful and would consolidate "current week" code between API, web, and PWA.
